### PR TITLE
fix a broken link

### PR DESCRIPTION
**Pull request by ktarbet**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@ktarbet : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/how-can-i-find-out-if-my-pointers-corrupt-a-memory-address-q.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ ms.workload:
 
 #### Check for heap corruption
 
-- Most memory corruption is actually due to heap corruption. Try using the Global Flags Utility (gflags.exe) or pageheap.exe. See [http://support.microsoft.com/default.aspx?scid=kb;en-us;286470](http://support.microsoft.com/default.aspx?scid=kb;en-us;286470).
+- Most memory corruption is actually due to heap corruption. Try using the Global Flags Utility (gflags.exe) or pageheap.exe. See [https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/gflags-and-pageheap](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/gflags-and-pageheap).
```
Comment (Mikejo5000) [inline docs/debugger/how-can-i-find-out-if-my-pointers-corrupt-a-memory-address-q.md]:
```suggestion
- Most memory corruption is actually due to heap corruption. Try using the Global Flags Utility (gflags.exe) or pageheap.exe. See [https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/gflags-and-pageheap](/windows-hardware/drivers/debugger/gflags-and-pageheap).
```

Comment (Mikejo5000) [reply docs/debugger/how-can-i-find-out-if-my-pointers-corrupt-a-memory-address-q.md]: @ktarbet If we use a relative link, we can avoid including the locale (en-us).

**Comment by PRMerger15** [comment]
@ktarbet : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**